docs/pages/foundations/motion/guidelines.tsx

```diff
@@ -0,0 +1,285 @@
+              <Markdown
+                text={`
+**Function:** cubic-bezier(0; 0; 1; 1;)
+**Recommended duration:**
+**Usage:** for properties unrelated to motion (like color and opacity). For example, a background color of a button that expands or changes upon hover can happen _**linearly**_ as it provides feedback to the user about their cursor position.
+
+**Usage examples:** Toggles, Checkboxes, Buttons, Text field
+        `}
+              />
+            </Flex.Item>
+          </Flex>
+
+          <Heading size="300">Examples in product</Heading>
+          <Flex gap={8} justifyContent="center" maxWidth={DOCS_COPY_MAX_WIDTH_PX}>
+            <Image height={542} src="/motion-gifs/radio-button.gif" width={250} />
+            <Image height={542} src="/motion-gifs/follow-button.gif" width={250} />
+          </Flex>
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection
+        description={`
+**Duration** is the time taken to complete any transition, interaction, and animation.
+Transitions should not be too fast or too slow, as this can make users feel rushed or impatient. To achieve smooth and responsive transitions, it is important to find the right balance of duration and easing.
+
+Durations are chosen based on these criteria:
+`}
+        name="Durations"
+      >
+        <Box maxWidth={DOCS_COPY_MAX_WIDTH_PX}>
+          <Markdown
+            text={`
+*   **Transition size** (that are based on the distance or the size of the object or element)
+    * Transitions covering smaller screen areas are assigned shorter durations, while those spanning larger areas have longer durations. Scaling the duration based on the transition area size ensures a consistent perception of speed.
+`}
+          />
+          <Markdown
+            text={`
+*   **Enter vs. exit transitions** (enter should be longer than exit)
+
+    * Shorter durations are applied to transitions that exit, dismiss, or collapse an element, as they require less user attention due to being swift. 
+
+    * On the other hand, enter transitions use longer durations, helping users in directing their attention on new elements appearing on the screen.
+`}
+          />
+        </Box>
+      </MainSection>
+
+      <MainSection
+        description={`
+        Animations should never cause a user harm or distract from a necessary task. However, inaccessible animations have the potential to cause seizures, motion sickness, or anxiety in users, so we must ensure we are creating accessible animations every time.
+
+        Below are 3 key considerations for accessibility. Note: these do not apply for “essential” animations, where [essential is defined](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html#essentialdef) as any animation where “if removed, would fundamentally change the information or functionality of the content, _and_ information and functionality cannot be achieved in another way that would conform.”
```

Comment:
This is referring to the WCAG 2.0 guidelines, let's update to WCAG 2.2: https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide.html#dfn-essential